### DISCUSSION TITLE
Where to post issues with the GUI or manuals?

### DISCUSSION THREAD

**Discussion by MitchHeineman** [USEPA/Stormwater-Management-Model#126] [General]
I've posted various questions about the GUI (and one or two about the manuals) in the Issues section, but this repository is presumably dedicated to the computational engine. Is there or should there be another place to discuss such matters?

**Comment by michaeltryby** [comment]
Hey @MitchHeineman, For the time being, this forum is appropriate for posting those issues. We do plan on gradually migrating more SWMM assets like the UI and documentation to GitHub. Please be patient while we get organized. Thank you!

**Comment by MitchHeineman** [comment]
EPA created https://github.com/USEPA/SWMM-GUI in July
